wpeverest/everest-forms
# Feature - Admin approval entries

### All Submissions:



### Changes proposed in this Pull Request:
Previously, entries were directly approved. This PR adds feature to either approve or deny the entries by the admin.  Admin can also delete the pending entries which has exceeded the limited days.

Closes # .

### How to test the changes in this Pull Request:

1. Go to Global Settings> General
2. Enable the toggle button for Enable the admin approval entries
3. Choose the email notification type either site admin or the custom email.
4. Enter the email subject and email body.
5. Check on the Automatically delete the pending approval entries to enable automatically delete the pending entries.
6. Click on Save Settings
7. Now, go to Form Settings > General Settings > Enable Admin Approval Entries
9. Click on the save button.

### Types of changes:



### Other information:




### Changelog entry

> Feature - Admin approval entries
